code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -142,74 +149,85 @@ Credit where due:
 /datum/game_mode/clockwork_cult/proc/greet_servant(mob/M) //Description of their role
 	if(!M)
 		return 0
-	var/greeting_text = "<br><b><span class='large_brass'>You are a servant of Ratvar, the Clockwork Justiciar.</span>\n\
-	Rusting eternally in the Celestial Derelict, Ratvar has formed a covenant of mortals, with you as one of its members. As one of the Justiciar's servants, you are to work to the best of your \
-	ability to assist in completion of His agenda. You may not know the specifics of how to do so, but luckily you have a vessel to help you learn.</b>"
+	var/greeting_text = "<br><span class='large_brass bold'>You are a servant of Ratvar!</span>\n\
+	<i>You have [round(time_to_prepare)] minutes to prepare before the crew is alerted of your presence and portals open across [station_name()].</i>\n\
+	<i>Need help? Look inside the box on your belt for an item called a clockwork slab, and click the Recollection button in the top left."
 	to_chat(M, greeting_text)
-	M.playsound_local('sound/ambience/antag/ClockCultAlr.ogg',100,0)
+	to_chat(M, sound('sound/ambience/antag/ClockCultAlr.ogg'))
 	return 1
 
-/datum/game_mode/proc/equip_servant(mob/living/L) //Grants a clockwork slab to the mob, with one of each component
+/datum/game_mode/proc/equip_servant(mob/living/carbon/human/L) //Grants a clockwork slab to the mob, with one of each component
 	if(!L || !istype(L))
-		return FALSE
-	var/obj/item/clockwork/slab/starter/S = new/obj/item/clockwork/slab/starter(null) //start it off in null
-	var/slot = "At your feet"
-	var/list/slots = list("In your left pocket" = slot_l_store, "In your right pocket" = slot_r_store, "In your backpack" = slot_in_backpack, "On your belt" = slot_belt)
-	if(ishuman(L))
-		var/mob/living/carbon/human/H = L
-		slot = H.equip_in_one_of_slots(S, slots)
-		if(slot == "In your backpack")
-			slot = "In your [H.back.name]"
-	if(slot == "At your feet")
-		if(!S.forceMove(get_turf(L)))
-			qdel(S)
-	if(S && !QDELETED(S))
-		to_chat(L, "<b>[slot] is a link to the halls of Reebe and your master. You may use it to perform many tasks, but also become oriented with the workings of Ratvar and how to best complete your \
-		tasks. This clockwork slab will be instrumental in your triumph. Remember: you can speak discreetly with your fellow servants by using the <span class='brass'>Hierophant Network</span> action button, \
-		and you can find a concise tutorial by using the slab in-hand and selecting Recollection.</b>")
-		to_chat(L, "<i>Alternatively, check out the wiki page at </i><b>https://tgstation13.org/wiki/Clockwork_Cult</b><i>, which contains additional information.</i>")
+		return
+	L.set_species(/datum/species/human)
```

Comment:
I'm not really giving a whole lot of thought to the whole reason they're there; the only reason they're forcefully humans is because of gameplay reasons. There's a lot of fluff you can put behind it, but I'd like to leave it up the players.

I do want the starting cultists to be normal humans, as stealth is important to them boarding the station. I could make them resemble humans at a glance but certain analyzers show that they're not, though, if I really wanted to go that route.